src/datoso_plugin_internetarchive/ia.py

```diff
@@ -26,7 +26,7 @@ def get_item(self):
 
     def get_download_path(self):
         """ Return the download path for files in InternetArchive item. """
-        self.path = f"https://{self.item.item_metadata['d1']}{self.item.item_metadata['dir']}"
+        self.path = f"https://{self.item.item_metadata['d1']}{self.item.item_metadata['dir']}/"
```

Comment:
Although is not needed anymore with the new `urljoin` implementation I think it is good to point out that the result is a directory. 👍

Reply:
Agreed. It was necessary for the purposes of using urllib.parse.join, hence its inclusion, however since it's a non-breaking change (in that the slash is stripped if necessary by the join function) it's better to leave it in this changed state as it provides more context in the relevance of the download path.